chia/full_node/full_node.py

```diff
@@ -1212,8 +1206,10 @@ async def respond_block(
             if block.transactions_info is not None
             else ""
         )
+        validation_complete = time.time()
```

Comment:
the other part of this change is to extend the timings to log. No surprise; this function is dominated by full block body validation, but now it also includes the pre-validation and the post-processing. I think it's important to include the post-processing because it seems that sometimes in involves sending messages to peers (which we probably shouldn't be doing while holding the blockchain mutex)